src/Routes.php

```diff
@@ -161,9 +163,21 @@ public function endpoints(): array
             return $endpoints;
         }
 
-        return array_filter($endpoints, function($endpoint) {
-            return in_array($endpoint['type'], $this->types);
-        });
+        if (isset($this->types['only']) and count($this->types['only'])) {
+            $only = $this->types['only'];
+            $endpoints = array_filter($endpoints, function($endpoint) use ($only) {
+                return in_array($endpoint['type'], $only);
+            });
+        }
+
+        if (isset($this->types['except']) and count($this->types['except'])) {
+            $except = $this->types['except'];
+            $endpoints = array_filter($endpoints, function($endpoint) use ($except) {
+                return !in_array($endpoint['type'], $except);
```

Comment:
```php
return in_array($endpoint['type'], $this->types['except']);
```

Reply:
I already did `$except = $this->types['except'];` on line number 174. so I think this one is good.